test/units/galaxy/test_role_install.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import os
+import pytest
+import tempfile
+
+from io import StringIO
+from units.compat.mock import MagicMock
```

Comment:
The `mocker` fixture is available in our tests. You can use `mocker.MagicMock` instead of importing it.
```suggestion
```